`<complex>` Enable users to disable normalization for division too. (#474)
NVIDIA/libcudacxx: docs/standard_api/numerics_library/complex.md

User-defined floating-point literals must be specified in terms of
  `long double`, so they lead to warns that are unable to be suppressed.

## Customizations

> Our implementation by default normalizes denormals during multiplication and division. This adds a significant runtime overhead, so we allow the user to disable that normalization if it is not desired.

Author: We should use `canonicalization` instead of normalization

> A user can define `LIBCUDACXX_ENABLE_SIMPLIFIED_COMPLEX_OPERATIONS` to disable normalization for both multiplication *and* division.

Author: ```suggestion
Definition of `LIBCUDACXX_ENABLE_SIMPLIFIED_COMPLEX_OPERATIONS` disables canonicalization for both multiplication *and* division.
```

> A user can also define `LIBCUDACXX_ENABLE_SIMPLIFIED_COMPLEX_DIVISION` or `LIBCUDACXX_ENABLE_SIMPLIFIED_COMPLEX_DIVISION` to disable normalization for multiplication or division individualy.

Author: ```suggestion
Definition of `LIBCUDACXX_ENABLE_SIMPLIFIED_COMPLEX_DIVISION` or `LIBCUDACXX_ENABLE_SIMPLIFIED_COMPLEX_DIVISION` disables canonicalization for multiplication or division individually.
```
